docker/test/run-test

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname $0)
+ABSOLUTE_PATH=$(realpath $HERE)
+
+$HERE/setup-test-repo
+
+docker network create orderly.runner_network
+
+docker run --rm -d \
+       --net=orderly.runner_network \
+       --name=orderly.runner_redis \
+       -p 127.0.0.1:6379:6379 \
+       redis
+
+docker volume create orderly-root-volume
+
+docker run --rm -d --pull=missing \
```

Comment:
that one is for a couple things:
1. you need a volume attached to a container to do anything with it so for us modifying the orderly repo, we could use this as the place to change things in the volume for testing
2. to view how things change in the working directory (since workers will be running things and producing result files and metadata of the run) theres a helper script `docker/test/copy-orderly-root` that will copy the folder so we can see it easily